react/features/filmstrip/reducer.js

```diff
@@ -55,6 +63,12 @@ ReducerRegistry.register(
                 ...state,
                 visible: action.visible
             };
+
+        case SET_FILMSTRIP_VISIBLE_PARTICIPANT_IDS:
+            return {
+                ...state,
+                visibleParticipantIds: action.visibleParticipantIds
```

Comment:
should we copy the array to make sure what remains in redux is immutable from the outside?